### ISSUE TITLE
Merge installer repo into SDK

### ISSUE THREAD

**Issue by marcpopMSFT** [dotnet/sdk#24415]
- [ ] Copy over issues
- [x] Move the installer repo into a subfolder of SDK
- [x] Figure out how to make it so you can build just the SDK or just the installer side
- [x] Find a home for the installer build page
- [ ] Figure out the cmake requirement so sdk developers don't have to install cmake
- [x] Modify the CI to include the installer legs
- [x] Modify the installer legs to build SDK and tests first (adding 6-10 minutes of time to installer)
- [x] Archive installer repo

Updating as we're doing an initial review of this work for potentially being done it .net 9.

Other scenarios to consider
	• Installer download page and feed list
	• Loc build from release branch for SDK and main branch for installer
	• CPM enabled in SDK
	• Dependabot and fabric bot in SDK
	• Workloads are overwritten in the SDK repo (overlaysdkonlkg.targets)
	• How would this affect stage0 updates and TFM updates?
	• How does this merge impact SB and VMR?
	• SDK repo reruns tests, installer repo reruns builds

For local dev build time, I think we need installer to not do it's crossgen build during the plain build. That increases the build time by nearly 2 minutes and if merged into SDK, would be too much to take on. Only do this build during build -pack

**Comment by mmitche** [comment]
Yes, there would be no step 3 with these improvements.

**Comment by MiYanni** [comment]
I've updated the tasks in the description.
- Issues aren't finished being cleaned up. We have <100 to go through and either close or transfer.
- The CMake requirement has [a separate issue](https://github.com/dotnet/sdk/issues/40859) created for it.
- I marked "Archive installer repo" as complete since we can't actually archive it until all the servicing branches are out-of-support. For now, we've updated the default branch and have a readme that redirects to the SDK repo.
